src/controller/user.controller.js

```diff
@@ -0,0 +1,90 @@
+const userService = require('../services/user.service');
+
+const get = (req, res) => {
+  const users = userService.getAll();
+
+  if (!users) {
+    res.send([]);
+    res.sendStatus(404);
```

Comment:
You're trying to send a response twice, which is not allowed in Express.js. Once the response is sent, you cannot send it again. So, you should remove the line `res.send([]);` because it's unnecessary and may cause errors.